src/pool/srv_cli.c

```diff
@@ -67,7 +67,7 @@ dsc_pool_open(uuid_t pool_uuid, uuid_t poh_uuid, unsigned int flags,
 	pool->dp_capas = flags;
 
 	/** attach to the server group and initialize rsvc_client */
-	rc = dc_mgmt_sys_attach(NULL, &pool->dp_sys);
+	rc = dc_mgmt_sys_attach(NULL, false, &pool->dp_sys);
```

Comment:
because i didn't want to change the default in this case as it's not on the init path that im trying to improve.
so passing false just uses this original behavior.